landingpage/src/components/AlterConfigForName.tsx

```diff
@@ -14,10 +14,12 @@ type AlterOptionKey =
 const AlterConfigForName: React.FC<{
 	queryString: string;
 	currentOwner: string;
+	onInscriptionChange: (output: string[]) => void;
```

Comment:
For a raw string it is not clear what the unit/format is:
`onInscriptionChange: (ascii: string) => void;`